setup.py

```diff
@@ -23,12 +23,16 @@
     packages=setuptools.find_packages(),
     install_requires=install_requirements,
     setup_requires=['pytest-runner'],
-    tests_require=['pytest', 'vcrpy'],
+    tests_require=[
+        'pytest==4.2.1',
```

Comment:
You should use version ranges like:
`pytest>=4.2.1,<4.3.0`

That makes sure we don't install a version with breaking changes, but it still allows us to receive bug fixes, etc.

Reply:
corregido